packages/core/demo/data/line.ts

```diff
@@ -39,6 +72,44 @@ export const lineOptions = {
 	}
 };
 
+export const sparklineLoadingOptions = {
+	title: "Sparkline (loading)",
+	axes: {
+		bottom: {
+			visible: false,
+			title: "2019 Annual Sales Figures",
+			mapsTo: "date",
+			scaleType: "time"
+		},
+		left: {
+			visible: false,
+			mapsTo: "value",
+			title: "Conversion rate",
+			scaleType: "linear"
+		}
+	},
+	grid: {
+		x: {
+			enabled: false
+		},
+		y: {
+			enabled: false
+		}
+	},
+	legend: {
+		enabled: false
+	},
+	tooltip: {
+		enabled: false
+	},
+	points: {
+		enabled: false
+	},
+	data: {
+		sparklineLoading: true
```

Comment:
so I'm wondering why `sparklineLoading` would be a separate entity? why can't sparkline just read the `loading flag`? sounds like this loading behaviour can be used for both line & sparkline graphs? correct @jeanservaas?

Reply:
The `loading` entity and `sparklineLoading` entity should be different. If we set `loading` to true, skeleton chart will be displayed. Only if we set `sparklineLoading` to true, we will be able to see loading sparkline. Unless we no longer want to show skeleton for line chart then we can just use `loading` as the flag to check whether we want to display loading sparkline or not. What do you think? @theiliad

Reply:
I'll respond in the change request